polkadot/node/core/dispute-coordinator/src/import.rs

```diff
@@ -584,7 +597,7 @@ impl ImportResult {
 			}
 		}
 
-		let new_state = CandidateVoteState::new(votes, env, now);
+		let new_state = CandidateVoteState::new(votes, env, now, is_disabled);
```

Comment:
Why don't we make this part of the environment?

Reply:
you mean the `is_disabled` closure that looks up the global state of dispute coordinator?